src/components/SwipeThumb/index.js

```diff
@@ -207,8 +207,8 @@ const SwipeThumb = props => {
     backgroundColor,
     borderColor,
     width: animatedWidth,
-    ...railStyles,
     ...(enableReverseSwipe ? styles.containerRTL : styles.container),
+    ...railStyles,
```

Comment:
Hello @maxenceg , Thanks for the PR. 
My intention was not to let anyone set container properties wrongly which will stop the component function correctly. But now this change(fix) will give full control to developers. This is totally fine.